pkg/iac/adapters/terraform/azure/database/adapt.go

```diff
@@ -435,3 +484,42 @@ func adaptMSSQLExtendedAuditingPolicy(resource *terraform.Block) database.Extend
 		RetentionInDays: retentionInDaysVal,
 	}
 }
+
+func adaptActiveDirectoryAdministrator(resource *terraform.Block) database.ActiveDirectoryAdministrator {
+	loginAttr := resource.GetAttribute("login")
+	loginVal := loginAttr.AsStringValueOrDefault("", resource)
+
+	return database.ActiveDirectoryAdministrator{
+		Metadata: resource.GetMetadata(),
+		Login:    loginVal,
+	}
+}
+
+func adaptAzureADAdministratorBlock(block *terraform.Block) database.ActiveDirectoryAdministrator {
+	// The azuread_administrator block uses login_username attribute
+	loginAttr := block.GetAttribute("login_username")
+	if loginAttr.IsNil() {
+		// Fallback to login if login_username is not present
+		loginAttr = block.GetAttribute("login")
+	}
+	loginVal := loginAttr.AsStringValueOrDefault("", block)
```

Comment:
You can use the `GetFirstAttributeOf` method to get the first non-null attribute, just update your branch as it was added recently.
```suggestion
	loginVal := block.GetFirstAttributeOf("login_username", "login").AsStringValueOrDefault("", block)
```